### DISCUSSION TITLE
Following since date

### DISCUSSION THREAD

**Discussion by AvramSilb** [Chatterino/chatterino2#3005] [Q&A] [answered]
Hi, 
I need to  get the "following since" date of a chatter for an analysis bot. How do you guys get this info?

Thanks

Avram

**Comment by apa420** [comment, marked as answer]
https://dev.twitch.tv/docs/api/reference#get-users-follows
https://github.com/Chatterino/chatterino2/blob/d5add4673083ba8830850ef5a4e73d8d454503b8/src/providers/twitch/api/Helix.cpp#L145-L162
